Sources/BrowserServicesKit/SecureVault/SecureVaultManager.swift

```diff
@@ -53,11 +52,6 @@ public protocol SecureVaultManagerDelegate: SecureVaultErrorReporting {
                             promptUserWithGeneratedPassword password: String,
                             completionHandler: @escaping (Bool) -> Void)
 
-    func secureVaultManagerShouldAutomaticallyUpdateCredentialsWithoutUsername(_: SecureVaultManager,
```

Comment:
I still see one or both of these delegate methods implemented in the codebases on iOS / macOS and in MockSecureVaultManagerDelegate; it would good to get these cleaned up 🙂